server/routes/appointments/appointmentsController.ts

```diff
@@ -447,9 +447,18 @@ export default class AppointmentsController {
       this.interventionsService.getSentReferral(accessToken, referralId),
       this.interventionsService.getSupplierAssessment(accessToken, referralId),
     ])
-    const appointment = await this.getSupplierAssessmentAppointmentFromDraftOrService(req, res, supplierAssessment)
+    const appointment = await this.getSupplierAssessmentAppointmentFromDraftOrService(
+      req,
+      res,
+      supplierAssessment,
+      referralId
+    )
 
     if (appointment === null) {
```

Comment:
if this is a draft appointment don't throw an error as draft process already handles the error